app/index.html

```diff
@@ -49,11 +49,14 @@
       </div>
     </nav>
 
-    <div ng-view style="margin-top: 5%;"></div>
+    <div class="container">
+      <div ng-view style="margin-top: 2%;"></div>
```

Comment:
Do we still need a margin-top here? Inline styling is ugly ☹️

Reply:
yeah we need it, but moved it to the app.css